### ISSUE TITLE
为什么link prediction evalue的时候，准确率一直下降呢

### ISSUE THREAD

**Issue by wakupoo** [hwwang55/GraphGAN#14]

**Comment by fukien** [comment]
Me too, both generator and discriminator are decreasing in accuracy. It seems that the link prediction result is a little different from the paper.

**Comment by NemesisAmos** [comment]
I met this problem when training other GANs, my guess is that since the generator and discriminator both evoluted under training process, I can see the GAN successfully learned features from real data, when the accuracy starts to drop, it may means that generator can not dig more useful feature from real data, which means you need to adjust your model(for instance a new loss function) or add more data.

**Comment by 1066118757** [comment]
Me too.

**Comment by hwwang55** [comment]
Hi guys! I suggest decreasing the learning rate.

**Comment by 915914082** [comment]
I decreased the learning rate to 1e-4 for both the discriminator and the generator, and trained for 100 epochs.  The accuracy for G is about 51% and the accuracy for D is about 60%, which is still a little different from the paper. So I am wondering could anyone share the exact parameters to get the 80%+ accuracy? Thanks a lot.

**Comment by treenewbee0** [comment]
> I decreased the learning rate to 1e-4 for both the discriminator and the generator, and trained for 100 epochs. The accuracy for G is about 51% and the accuracy for D is about 60%, which is still a little different from the paper. So I am wondering could anyone share the exact parameters to get the 80%+ accuracy? Thanks a lot.

I set the learning rate to 1e-5 and I succeeded.It can be set to 1e-6 for greater accuracy, but it also requires more iterations than the default config.

**Comment by ZainZhao** [comment]
I think you can read the paper` CFGAN`, it seems to explan the reason of that

**Comment by tomatowithpotato** [comment]
> > I decreased the learning rate to 1e-4 for both the discriminator and the generator, and trained for 100 epochs. The accuracy for G is about 51% and the accuracy for D is about 60%, which is still a little different from the paper. So I am wondering could anyone share the exact parameters to get the 80%+ accuracy? Thanks a lot.
> 
> I set the learning rate to 1e-5 and I succeeded.It can be set to 1e-6 for greater accuracy, but it also requires more iterations than the default config.

so how many iterations you succeeded when set the learning rate to 1e-5? by default?

**Comment by zhangzef** [comment]
我把lr_gen和lr_dis设成1e-5，batch_size设置成1024，在epoch 30的时候gen的accuracy达到最大0.884，比论文中的accuracy高一些
